src/django_components/util/exception.py

```diff
@@ -0,0 +1,61 @@
+from contextlib import contextmanager
+from typing import Generator, List
```

Comment:
This file contains the logic for modifying error messages to include the component and slot path, e.g.

```
 KeyError: "An error occured while rendering components MyPage > MyComponent > MyComponent(slot:content) > Broken
```

This was previously defined in [`Component._render()`](https://github.com/django-components/django-components/pull/936/files#diff-35c84ebceae50f7149f03bbcaf2b4f1289bc2984bf7859ca521b5dbe7960c396L931).